tests/static_lib_arch_flags/app_static_lib_srcs_flags_archive0_xs2/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.21)
 include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
-project(static_lib_srcs_flags_archive0_xs2)
+project(static_lib_arch_flags_archive0_xs2)
```

Comment:
Looks like the app directory needs renaming too, ideally.

Reply:
Renamed app_static_lib_srcs_flags_* -> app_static_arch_flags_*